src/main/java/pl/edu/zut/mad/schedule/exception/NotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package pl.edu.zut.mad.schedule.exception;
+
+public class NotFoundException extends RuntimeException {
+
+    private final int id;
+
+    public NotFoundException(int id) {
```

Comment:
Name parameter `id` is little confusing for me because at this time I see only usage with `albumNumber`. The reason is this will be used with other repositories?

Reply:
It might be used in future for other objects, This exception shouldn't be too specific.